sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationClientLiveTests.cs

```diff
@@ -82,6 +82,11 @@ public async Task StartBuildModel(bool singlePage)
         [RecordedTest]
         public async Task StartBuildModelWithNeuralBuildMode()
         {
+            if (Recording.Mode == RecordedTestMode.Live)
```

Comment:
Would you be so kind as to add a comment here just noting why we're suppressing only in Live?  I realize that you linked the issue, but having a local summary is helpful.

Reply:
Done!